.rubocop_todo.yml

```diff
@@ -6,11 +6,12 @@
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 3
+# Offense count: 8
 # Configuration parameters: CountComments, ExcludedMethods.
 # ExcludedMethods: refine
 Metrics/BlockLength:
-  Max: 151
+  Exclude:
+    - 'spec/**/*'
```

Comment:
Should this be in `.rubocop.yml` rather than `.rubocop_todo.yml`?

Reply:
This seems unlikely to be a thing we will want to correct, so I'll shift it over.